modules/game/server/sockets/game.server.socket.config.js

```diff
@@ -422,11 +422,20 @@ module.exports = function (io, socket) {
     var game = games[room];
 
     if (!game.Game.started && username === game.Game.getHost()) {
-      game.prompts = TopicList.getTopicWords(game.Game.topicListName, game.Game.topicListDifficulty);
-      ServerUtils.shuffleWords(game.prompts);
-      game.Game.startGame();
-      io.to(room).emit('startGame');
-      startRound(game, room);
+      topicsHelper.getWords(game.Game.topicListName, function (err, words) {
```

Comment:
Read from the database instead of the static list. This is on the server-side when starting a game.